src/WinRT.Interop.Generator/Factories/InteropMethodDefinitionFactory.IObservableMap2Impl.cs

```diff
@@ -0,0 +1,278 @@
+            TypeSignature eventHandlerType = interopReferences.MapChangedEventHandler2.MakeGenericReferenceType(keyType, valueType);
+
+            // Prepare the 'EventRegistrationTokenTable<MapChangedEventHandler<KEY_TYPE>, <VALUE_TYPE>>' signature
+            TypeSignature eventRegistrationTokenTableType = interopReferences.EventRegistrationTokenTable1.MakeGenericReferenceType(eventHandlerType);
+
+            // Prepare the 'ConditionalWeakTable<<MAP_TYPE>, EventRegistrationTokenTable<MapChangedEventHandler<<KEY_TYPE>, <VALUE_TYPE>>>' signature
+            TypeSignature conditionalWeakTableType = interopReferences.ConditionalWeakTable2.MakeGenericReferenceType(
+                mapType,
+                interopReferences.EventRegistrationTokenTable1.MakeGenericReferenceType(eventHandlerType));
+
+            // Define the 'remove_MapChanged' method as follows:
+            //
+            // [UnmanagedCallersOnly(CallConvs = [typeof(CallConvMemberFunction)])]
+            // private static int remove_MapChanged(void* thisPtr, EventRegistrationToken token)
+            MethodDefinition remove_MapChangedMethod = new(
+                name: "remove_MapChanged"u8,
+                attributes: MethodAttributes.Private | MethodAttributes.HideBySig | MethodAttributes.Static,
+                signature: MethodSignature.CreateStatic(
+                    returnType: module.CorLibTypeFactory.Int32,
+                    parameterTypes: [
+                        module.CorLibTypeFactory.Void.MakePointerType(),
+                        interopReferences.EventRegistrationToken.ToValueTypeSignature().Import(module)]))
+            {
+                CustomAttributes = { InteropCustomAttributeFactory.UnmanagedCallersOnly(interopReferences, module) }
+            };
+
+            // Jump labels
+            CilInstruction ldarg_0_tryStart = new(Ldarg_0);
+            CilInstruction ldc_i4_0_return0 = new(Ldc_I4_0);
+            CilInstruction callvirt_catchHResult = new(Callvirt, interopReferences.Exceptionget_HResult.Import(module));
+            CilInstruction ldloc_3_returnHResult = new(Ldloc_3);
+
+            // Declare the local variables:
+            //   [0]: '<MAP_TYPE>' (the 'unboxedValue' object)
+            //   [1]: 'EventRegistrationTokenTable<MapChangedEventHandler<<KEY_TYPE>, <VALUE_TYPE>>>' (the 'table' object)
+            //   [2]: 'MapChangedEventHandler<<KEY_TYPE>, <VALUE_TYPE>>' (the 'managedHandler' object)
+            //   [3]: 'int' (the 'HRESULT' to return)
+            CilLocalVariable loc_0_unboxedValue = new(mapType.Import(module));
+            CilLocalVariable loc_1_table = new(eventRegistrationTokenTableType.Import(module));
+            CilLocalVariable loc_2_managedHandler = new(eventHandlerType.Import(module));
+            CilLocalVariable loc_3_hresult = new(module.CorLibTypeFactory.Int32);
+
+            // Create a method body for the 'add_MapChanged' method
```

Comment:
remove?

Reply:
Same copy-paste error, fixed, thank you!